src/modules/cpu/bsd.cpp

```diff
@@ -95,3 +95,7 @@ std::vector<std::tuple<size_t, size_t>> waybar::modules::Cpu::parseCpuinfo() {
   free(cp_time);
   return cpuinfo;
 }
+
+std::vector<float> waybar::modules::Cpu::parseCpuFrequencies() {
+  throw std::runtime_error("Cpu frequency is not implemented on BSD.");
```

Comment:
Try turning error into warning and maybe replace zero with something easy to notice e.g.,
```c++
#include <spdlog/spdlog.h>
#include <cmath>              // INFINITY
```
```suggestion
  static std::vector<float> frequencies;
  if (frequencies.empty()) {
    spdlog::error("cpu/bsd: parseCpuFrequencies is not implemented");
    frequencies.push_back(INFINITY);
  }
  return frequencies;
```